### PR TITLE
Upgrade omise-node api to v2019

### PR DESCRIPTION
**Pull request by gaurab13**
## Description
- Upgrade library to support API version 2019-05-29

### More information (if any) 

- Added setup to run the typescript examples
- Refactored example scripts
- Updated resource fields according to api v 2019-05-29

### Related links:

- Issue ID: <github-issue-id>

## Decision(s)

In the context of < use case / user story >, facing < concern > we decided for < option > to achieve < quality / goal >, accepting < downside >.

***NOTE: If the change introduces a new feature flag, please mention:  ***
  - ***When / Conditions to remove feature flag***
  - ***Please add label "FeatureFlag" to feature flag removal story***

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,18 +26,21 @@
   "homepage": "https://github.com/omise/omise-node",
   "dependencies": {
     "@types/node": "^8.10.66",
+    "dotenv": "^16.4.5",
```
Comment (aashishgurung) [inline package.json:29]: Is this suppose to be here or under dev dependencies?

Comment (gaurab13) [reply package.json:29]: Should be inside dev, i will update it.

**Review by mmkittisak — APPROVED** [review]

**Review by muthuswamyopn — APPROVED** [review]
Approving the readme. 2 minor comments.

**Some examples of library usage are also listed below:**

Rewrite as:

**Some examples of library usage are also listed as follows:**

--

Maintenance mode means bug fixes and security patches only**.**

The full stop is missing in the text.

**Review by muthuswamyopn — APPROVED** [review]
Readme is fine.

**Review by aashishgurung — APPROVED** [review]

**Review by muthuswamyopn — APPROVED** [review]
Approving the Readme.

**Pull request merged**